testdata/scripts/literals.txt

```diff
@@ -24,6 +25,23 @@ rm main$exe
 garble -literals build
 bincmp main$exe main_old$exe
 
+# Also check that the binary is true random.
+rm main$exe
+garble -literals -debugdir=.obf-src -seed=8J+Ri/Cfh6fwn4e+ build
```

Comment:
If you want true random use -seed=random

Reply:
I want to make sure the binary is different. I'm not sure random is a good solution.

Reply:
I'll replace the comment with "Also check that the binary is different from previous builds" so there's no confusion.

Reply:
Ok